packages/client/wallets/aa/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@crossmint/client-sdk-aa",
+    "version": "0.0.9-alpha.6",
+    "author": "Paella Labs Inc",
```

Comment:
Paella inc

Reply:
do u want me to change all the packages we have? they all use this same author

Reply:
Mind filing a task and we do it later? But yes, this is the wrong company XDDD